### DISCUSSION TITLE
DateFormat

### DISCUSSION THREAD

**Discussion by JanSkovhus** [totaldebug/atomic-calendar-revive#1466] [Support] [answered]
How can i show the date in a format that looks like this (dddd D MMM) ?
By me it looks like this.
![image](https://github.com/totaldebug/atomic-calendar-revive/assets/60315341/84135595-bb50-4c84-8a5b-eb6ff96e3a45)

Regards Jan

**Comment by marksie1988** [comment, marked as answer]
At this time it's not possible.
